### ISSUE TITLE
Zephrus G15 Support

### ISSUE THREAD

**Issue by jbrgilbrech** [aredden/electron-G14Control#97]
I was wondering, is there any future support with the GA503QR?

**Comment by kaizenblumenblupper** [comment]
Having the same interest. Hope it does (it's a shame if it doesn't)

**Comment by jbrgilbrech** [comment]
I agree, I hope there is some support since we can all agree Armory Crate is terrible.

**Comment by varad-kadam** [comment]
I'm casting my vote as well!
It's be wonderful to have proper support of the G15 as well.
